### ISSUE TITLE
Booleans do not round-trip with JSON::PP or Mojo::JSON

### ISSUE THREAD

**Issue by oschwald** [rurban/Cpanel-JSON-XS#40]
A boolean from this modules produces `encountered object '1', but neither allow_blessed nor convert_blessed settings are enabled` with `JSON::PP` and a 0/1 with `Mojo::JSON`. Both modules [[1](https://metacpan.org/source/MAKAMAKA/JSON-PP-2.27300/lib/JSON/PP.pm#L416), [2](https://github.com/kraih/mojo/blob/master/lib/Mojo/JSON.pm#L241)] currently use and expect a `JSON::PP::Boolean` blessed scalar reference, while this module produces a `JSON::XS::Boolean`. New versions of `JSON::XS` also [appear to produce a `JSON::PP::Boolean`](https://metacpan.org/source/MLEHMANN/Types-Serialiser-1.0/Serialiser.pm#L105).

Example:

```perl
use v5.16;

use Cpanel::JSON::XS qw( decode_json );
use JSON::PP qw( encode_json );

say encode_json(decode_json('[true]'));
```

**Comment by oschwald** [comment]
Thanks! :+1: This seems to work well for me.

**Comment by rurban** [comment]
Merged into master for the next release 3.0201
